sync_data/actblue_mappings/production.py

```diff
@@ -19,6 +19,16 @@
     'lineitems#committeeName': 'field_971',
 }
 
+KNACK_DONOR_REQUIRED_FIELDS = [
+    'field_897', # first name
```

Comment:
These field numbers should match the field numbers in the above ACT_BLUE_MAPPING_SCALARS dict